### PR TITLE
Fix Expert Mode Switch

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: https://github.com/betaflight/betaflight-configurator/issues/2552

The local Expert Mode switch in firmware_flasher is set by the global Expert Mode switch (when connected on any tab) but didn't trigger an update on the global switch after a change.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
I was not aware of that... we have two different expert modes?

**Comment by haslinghuis** [comment]
Yes it's confusing. Going to look at it again.

**Comment by haslinghuis** [comment]
Thanks @McGiverGim - we simply needed to trigger the change. Now both expert mode switches (on firmware_flasher and when connected) are working again as one switch and are hopefully less confusing. When a user changes the mode its as long as Configurator is running separate from the permanentExpertMode switch in option tab.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Comment by Asizon** [comment]
Intereseting issue to come back to these expert functions. Seems that `Permanently enable expert mode` function in options tab is not working as the name says, I think that if it is permanently we should not give the user options to deactivate them in another way, since it also has problems with the current code. Maybe lock firmware flasher selector in expert mode and hidde top expert mode switch?

**Comment by haslinghuis** [comment]
Agree, if permanent export mode is enabled we should disable selector in firmware flasher and in top bar.

**Review by mikeller — APPROVED** [review]
Interesting find - I always thought it was meant to work the way it does now. But it's clearly a bug. :grinning:

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
